inscriptions/ISic0754.xml

```diff
@@ -173,12 +177,11 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:lang="la"><!-- to edit and encode -->
+            <div type="edition" xml:lang="la">
                 <ab>
-                    <lb n="1"/>[--provide]ntia
-                    <lb n="2"/>[--v(iri)] p(erfectissimi) praef(ecti) ann(onae)
-                    <lb n="3"/>[idemq(ue) v(ices) a(gentis) praef(ectorum) pr]aetorio (duorum) e(minentissimorum)
-                    v(irorum)
+                    <lb n="1"/><gap reason="lost" extent="unknown" unit="character"/> <supplied reason="lost">provide</supplied>ntia
+                    <lb n="2"/><gap reason="lost" extent="unknown" unit="character"/> <supplied reason="lost"><expan><abbr>v</abbr><ex>iri</ex></expan></supplied> <expan><abbr>p</abbr><ex>erfectissimi</ex></expan> <g type="interpunct">·</g> <expan><abbr>praef</abbr><ex>ecti</ex></expan> <g type="interpunct">·</g> <expan><abbr><hi rend="superscript">a</hi>nn</abbr><ex>onae</ex></expan>
+                    <lb n="3"/><supplied reason="lost"><expan><abbr>idemq</abbr><ex>ue</ex></expan> <expan><abbr>v</abbr><ex>ices</ex></expan> <expan><abbr>a</abbr><ex>gentis</ex></expan> <expan><abbr>praef</abbr><ex>ectorum</ex></expan> pr</supplied>aetorio <supplied reason="omitted">duorum</supplied> <expan><abbr><hi rend="superscript">e</hi><hi rend="superscript"><am>e</am></hi><hi rend="superscript">m</hi><hi rend="superscript"><am>m</am></hi></abbr><ex>intissiorum</ex></expan> <g type="interpunct">·</g> <expan><abbr><hi rend="superscript">v</hi><hi rend="superscript"><am>v</am></hi><ex>irorum</ex></abbr></expan>
```

Comment:
I think this should be `<hi rend="supraline">` and not superscript? Also the (duorum) is part of the abbreviation expansion, so should go within the expansion of virorum in my view, rather than supplied="omitted"

Reply:
Fixed